### ISSUE TITLE
A11y_VirtualAgentBots_VirtualAgentHome_ChatDialog_Screenreader: Using narrator the form landmark is announced twice to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/BotFramework-WebChat#2444]
**Environment Details:**
Browser Version: Microsoft Edge 44.18362.267.0
Windows Version: Windows10
Screen reader: Narrator, NVDA (2019.2)\
MAS Violated: MAS 1.3.1

**Steps to Reproduce:**
1. Home page should open for Virtual Agent Bots.
2. Navigate to main content area chat dialog,  the left side of the screen.
3. Start screen reader.
4. Navigate to the "Chat" section and then navigate to the "Type your message" edit field.
5. Listen to what all information is announced.

**Actual:**
When screen reader users navigate to the "Type your message" edit the form landmark is announced twice. It is announced as "form landmark status form landmark Type your message edit" which is incorrect.

**This issue is observed only in Narrator and not in NVDA.**

**Expected:**
The form landmark should be announced just once for the screen reader users while navigating through the "Type your message" edit field.

**User Impact:**
If incorrect information is conveyed to the screen reader users then they will confused and might miss out some functionality.

**Comment by corinagum** [comment]
this should be fixed in https://github.com/microsoft/BotFramework-WebChat/pull/2423
needs verification and close.

**Comment by cwhitten** [comment]
Thank you the issue. However, we are in the midst of an internal a11y audit to meet WCAG 2.1. The mechanisms to open, track, and address issues uncovered during the audit is a different process. I will close this for now.
